lib/core/collapsable/accordion-group.component.ts

```diff
@@ -82,15 +82,15 @@ export class AccordionGroupComponent implements AfterViewInit {
     }
 
     hasHeadingIcon() {
-        return this.headingIcon ? true : false;
+        return !!this.headingIcon;
     }
 
     onHeaderClick(): void {
         this.headingClick.emit(this.heading);
     }
 
-    isExpandable(event: any) {
```

Comment:
the prefix "is" means the method should only return the status not call a method like 
expandPanel